README.md

```diff
@@ -22,9 +22,14 @@ Ailly's best feature is rapidly itearting on prompt engineering. By keeping your
 
 These properties can be set in a combination of places, includeing the command line, .aillyrc, and greymatter. Later settings override earlier.
 
-* **`combined`** `boolean` If true, the file's body is the response and the prompt is in the greymatter key `prompt`. If false, the file's body is the prompt and the response is in `{file_name}.ailly.md`. Default false.
-* **`skip`** `boolean` If true, the prompt will not be sent through the LLM (but it will be part of the context).
-* **`isolated`** `boolean` If true, the LLM inference will only include the system prompt, and not the prior context in this folder.
+- **`combined`** `boolean` If true, the file's body is the response and the prompt is in the greymatter key `prompt`. If false, the file's body is the prompt and the response is in `{file_name}.ailly.md`. Default false.
+- **`skip`** `boolean` If true, the prompt will not be sent through the LLM (but it will be part of the context).
+- **`isolated`** `boolean` If true, the LLM inference will only include the system prompt, and not the prior context in this folder.
+- **`context`** `content` | `folder` | `none`
+  - `content` (default) loads files from the root folder and includes them alphabetically, chatbot history style, before the current file when generating.
```

Comment:
I don't love the name `content`, it's too vague. The difference between this and `folder` is unclear.

Reply:
Suggestions?

Reply:
What we're really doing is identifying if we want to treat a file as conversation or just additional context. My question, can ailly "just figure this out"?
If a folder contains only conversations or only non-ailly files, it seems a pretty straightforward yes.
If a folder contains a mix of both, it's less straightforward but I think it still works?
At that point, we don't need either of these flags and all we need to do is identify root.

Reply:
What about 'conversation'|'folder'|'none' for now, and if we can better infer the flag in the future we will?

Reply:
That works for me.

Reply:
Done.